sdccc/src/main/java/com/draeger/medical/sdccc/TestSuite.java

```diff
@@ -460,6 +461,23 @@ protected void defaultConfigure() {
             final var configModuleParser = new TomlConfigParser(TestSuiteConfig.class);
             configModuleParser.parseToml(configFileStream, configModule);
         }
+
+        // read multicast TTL from config
+        final var configInjector = Guice.createInjector(
+                Modules.override(new DefaultTestSuiteConfig()).with(configModule));
+
+        final var multicastTTL =
+                configInjector.getInstance(Key.get(Long.class, Names.named(TestSuiteConfig.NETWORK_MULTICAST_TTL)));
+
+        // configure SDC-ri to use the multicastTTL
+        final var sdcriConfigModule = new AbstractConfigurationModule() {
+
+            @Override
+            protected void defaultConfigure() {
+                bind(DpwsConfig.MULTICAST_TTL, Integer.class, multicastTTL.intValue());
```

Comment:
Why don't you bin DpwsConfig.MULTICAST_TTL in TestClientUtil.java?

Reply:
It is still necessary as otherwise the call to createInjector() at the end of the method fails.

Reply:
I disagree, you can remove all changes in TestSuite.java. In TestClientUtil.java you then pass NETWORK_MULTICAST_TTL instead of MULTICAST_TTL and create the binding in the constructor.

Reply:
Done. This even simplified things! Great Idea!